mungegithub/mungers/submit-queue.go

```diff
@@ -1636,7 +1651,7 @@ func (sq *SubmitQueue) serveMergeInfo(res http.ResponseWriter, req *http.Request
 	out.WriteString("PRs must meet the following set of conditions to be considered for automatic merging by the submit queue.")
 	out.WriteString("<ol>")
 	if gateCLA {
-		out.WriteString(fmt.Sprintf("<li>The PR must have the label %q or %q </li>", cncfClaYesLabel, claHumanLabel))
+		out.WriteString(fmt.Sprintf("<li>The PR must have the label %q </li>", fmt.Sprintf("%v", sq.ClaYesLabels)))
```

Comment:
This will quote the slice like `"[cncf-cla: yes cncf-cla: human approved]"`. Replace the nested `Sprintf` call with just `sq.ClaYesLabels` (but actually the reference you saved while holding the lock) to quote the elements of the slice instead of the slice itself.